### PR TITLE
Added Tooltips for item conduit upgrade slots

### PR DESCRIPTION
**Pull request by HenryLoenwind**

### CODE REVIEW

File: resources/assets/enderio/lang/en_US.lang [hunk 1]
```diff
@@ -873,6 +873,10 @@ enderio.gui.conduit.item.roundRobinEnabled=Round Robin Enabled
 enderio.gui.conduit.item.roundRobinDisabled=Round Robin Disabled
 enderio.gui.conduit.item.priority=Priority
 
+enderio.gui.conduit.item.filterupgrade=Insert Item Filter
+enderio.gui.conduit.item.speedupgrade=Insert Item Conduit Speed Up-/ Downgrade
```
Comment (tterrag1098) [inline resources/assets/enderio/lang/en_US.lang]: This formatting seems odd. Maybe "Insert Item Conduit Speed Upgrade/Downgrade"

**Comment by HenryLoenwind** [comment]
ok, but I made it "Upgrades/Downgrade", because you can add more than one upgrade

**Comment by tterrag1098** [comment]
Automerge is broken :(

**Comment by HenryLoenwind** [comment]
Should work now.

My bad, btw. Somehow the en_US.lang change ended up in another branch, too.

**Comment by tterrag1098** [comment]
Could you squash to one commit?

**Comment by HenryLoenwind** [comment]
You really want the merge squash in there?

**Pull request merged**